README.md

```diff
@@ -32,7 +32,7 @@ A developer embeds chat.com on two of their sites site-a.com and site-b.com. cha
 
 ### [Example 2](https://groups.google.com/a/chromium.org/g/blink-dev/c/24hK6DKJnqY/m/fybXzBdwCAAJ)
 
-This SaaS product has a heavy reliance on shared workers and this would break customer use cases.  Shared workers are used to coordinate Web RTC signaling and websocket management which is critical for the app. For example, the shared worker is used to support seamless multi-tab use cases and acts as a gatekeeper for managing audio and notifications if there are multiple instances of this app open (i.e., only a single tab can host an audio).
+This SaaS product has a heavy reliance on Broadcast Channel and this would break customer use cases. Broadcast Channel is used to coordinate Web RTC signaling and websocket management which is critical for the app. For example, the channel is used to support seamless multi-tab use cases and acts as a gatekeeper for managing audio and notifications if there are multiple instances of this app open (i.e., only a single tab can host an audio).
```

Comment:
Hmm the above link says Shared Workers though so I'm not sure we should replace that here. There's https://github.com/wanderview/quota-storage-partitioning/issues/9 if you're looking for a BroadcastChannel use case :)